iota/adapter/__init__.py

```diff
@@ -344,7 +328,7 @@ def get_uri(self):
     async def send_request(self, payload, **kwargs):
         # type: (dict, dict) -> dict
         kwargs.setdefault('headers', {})
-        for key, value in iteritems(self.DEFAULT_HEADERS):
+        for key, value in iter(self.DEFAULT_HEADERS.items()):
```

Comment:
```suggestion
        for key, value in self.DEFAULT_HEADERS.items():
```

(`dict.items()` returns an iterator)